app/controllers/posts_controller.rb

```diff
@@ -1,3 +1,5 @@
+require 'cancancan'
+
 class PostsController < ApplicationController
   def index
```

Comment:
**You did great job!**

- [ ] I tested your app ant it's working as expected. However, you also need to add `load_and_authorize_resource` rule on top of your PostsController class. Kindly add it and get approved :) Need to learn more about `load_and_authorize_resource` ckeck this [link](https://stackoverflow.com/questions/30840231/cancan-explanation-of-load-and-authorize-resource)